context_windows.go

```diff
@@ -0,0 +1,9 @@
+package continuity
+
+import "os"
+
+// getUidGidFromFileInfo extracts the user and group IDs from a fileinfo.
+// This is Unix-specific functionality.
```

Comment:
Do we not foresee putting `ACEs` in uid/gid? If not, can we revert these fields to integers?

On other systems, I have seen these as strings since they can be used for the ACE identifier, but I may be missing something here.